src/main/java/gov/usgs/locator/LocService.java

```diff
@@ -135,21 +130,21 @@ public LocOutput getLocation(final LocInput in) throws LocationException {
 
     // print input for debugging
     LOGGER.info("Input: \n" + event.getHydraInput(false));
-    
+
     // make sure we have a slab resolution
-    if(slabRes == null) {
-    	slabRes = "2spd";
+    if (in.SlabResolution == null) {
+      in.SlabResolution = "2spd";
```

Comment:
minor - should this be a constant (e.g. DEFAULTSLABRESOLUTION)

Reply:
Probably, but it's only in one place in the repo so it didn't seem worth it.